src/esp/gfx/RenderCamera.cpp

```diff
@@ -35,27 +32,11 @@ void RenderCamera::setProjectionMatrix(int width,
                                        float zfar,
                                        float hfov) {
   const float aspectRatio = static_cast<float>(width) / height;
-  camera_
-      ->setProjectionMatrix(Mn::Matrix4::perspectiveProjection(
-          Mn::Deg{hfov}, aspectRatio, znear, zfar))
+  MagnumCamera::setProjectionMatrix(
+      Mn::Matrix4::perspectiveProjection(Mn::Deg{hfov}, aspectRatio, znear,
+                                         zfar))
       .setViewport(Magnum::Vector2i(width, height));
 }
 
-mat4f RenderCamera::getProjectionMatrix() {
-  return Eigen::Map<mat4f>(camera_->projectionMatrix().data());
-}
-
-mat4f RenderCamera::getCameraMatrix() {
-  return Eigen::Map<mat4f>(camera_->cameraMatrix().data());
-}
-
-MagnumCamera& RenderCamera::getMagnumCamera() {
-  return *camera_;
-}
-
-void RenderCamera::draw(MagnumDrawableGroup& drawables) {
```

Comment:
Oh, I need this draw(...) function in derived class so that I can do culling within.

Can we keep it? If you keep it, add a comment, e.g.,
// TODO: visibility culling

```CPP
void RenderCamera::draw(MagnumDrawableGroup& drawables) {
    this->MagnumCamera::draw(drawables);
}
```